notifications_utils/recipients.py

```diff
@@ -448,11 +448,12 @@ def validate_email_address(email_address, column=None):  # noqa (C901 too comple
     if not match:
         raise InvalidEmailError
 
-    hostname = match.group(1)
-    # don't allow consecutive periods in domain names
-    if '..' in hostname:
+    # don't allow consecutive periods in either part
+    if '..' in email_address:
```

Comment:
Maybe use a regex like 

```python
r"\.{2,}"
```

to also catch more than 2 dots.

Reply:
this'll catch more than 2 dots - it returns True after the first time it sees two dots

Reply:
Based on this https://regexr.com/3rb59 it matches 2 or more dots which I think it's what we're after?

Reply:
ah this isn't a regex this is a simple python substring check. Should be more performant than a regex (worst case O(N))